src/queries/collectionById.graphql

```diff
@@ -3,12 +3,13 @@
 query collectionById($id: String!) {
   collectionEntity(id: $id) {
     ...collection
-    nfts {
+    nfts(filter: { burned: { distinctFrom: true } }) {
       nodes {
         id
         metadata
         name
         price
+        burned
```

Comment:
maybe reuse `nft` fragment ? (`nft.graphql`)

Reply:
I'm not sure what you mean here by "reuse ```nft```fragment"